java/pom.xml

```diff
@@ -0,0 +1,125 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+ <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.databricks.mlflow</groupId>
+  <artifactId>mlflow-java</artifactId>
+  <packaging>jar</packaging>
+  <version>0.4.2</version>
+  <name>mlflow-java</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+    <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>3.8.1</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.scala-lang</groupId>
+        <artifactId>scala-library</artifactId>
+        <version>2.11.12</version>
```

Comment:
Do we need to explicitly depend on a scala-library version? Is this because of MLeap? Wouldn't that bring this in for us?